vue/src/App.vue

```diff
@@ -0,0 +1,358 @@
+<script setup>
+import { onMounted, onUnmounted, ref } from "vue";
+
+onMounted(() => {
+  const script = document.createElement("script");
+  script.text = `
+    (function () {
+
+      // @NOTE: In order to load Web Chat, you have to set the PLATFORM_ID and DOMAIN in the script.
+      // https://developers.helpshift.com/web-chat/getting-started/#manual-embed
+
+      var PLATFORM_ID = "${import.meta.env.VITE_APP_PLATFORM_ID}",
+        DOMAIN = "${import.meta.env.VITE_APP_DOMAIN}",
+        LANGUAGE = "en";
+
+      // Do not modify the code below
+
+      // @NOTE: It's a good idea to set all widget options and information here.
+      // However, in the case where some information needs to be added later (async),
+      // use the updateHelpshiftConfig API https://developers.helpshift.com/web-chat/api/
+
+      window.helpshiftConfig = {
```

Comment:
[P3] - Same as previous PR. We can move this to a utility file if there is no vue specific code.